[2023-12-27T09:36:28.841+0545] {processor.py:161} INFO - Started process (PID=6823) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:36:28.842+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:36:28.842+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:36:28.842+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:36:28.846+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:36:28.845+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:36:28.846+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:36:28.861+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.021 seconds
[2023-12-27T09:37:19.258+0545] {processor.py:161} INFO - Started process (PID=6948) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:37:19.262+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:37:19.264+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:37:19.263+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:37:19.272+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:37:19.271+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:37:19.273+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:37:19.294+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.043 seconds
[2023-12-27T09:37:54.441+0545] {processor.py:161} INFO - Started process (PID=6993) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:37:54.444+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:37:54.445+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:37:54.445+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:37:54.451+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:37:54.450+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:37:54.452+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:37:54.475+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.039 seconds
[2023-12-27T09:38:31.666+0545] {processor.py:161} INFO - Started process (PID=7044) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:38:31.669+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:38:31.670+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:38:31.669+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:38:31.674+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:38:31.673+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:38:31.675+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:38:31.692+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.029 seconds
[2023-12-27T09:39:09.598+0545] {processor.py:161} INFO - Started process (PID=7135) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:39:09.601+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:39:09.603+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:39:09.602+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:39:09.609+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:39:09.608+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:39:09.610+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:39:09.632+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.039 seconds
[2023-12-27T09:39:45.627+0545] {processor.py:161} INFO - Started process (PID=7177) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:39:45.630+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:39:45.631+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:39:45.630+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:39:45.635+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:39:45.634+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:39:45.635+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:39:45.651+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.027 seconds
[2023-12-27T09:40:05.575+0545] {processor.py:161} INFO - Started process (PID=7228) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:40:05.576+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:40:05.577+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:40:05.577+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:40:05.584+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:40:05.583+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:40:05.584+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:40:05.611+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.040 seconds
[2023-12-27T09:40:43.917+0545] {processor.py:161} INFO - Started process (PID=7306) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:40:43.920+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:40:43.923+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:40:43.922+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:40:43.931+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:40:43.929+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:40:43.932+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:40:43.952+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.043 seconds
[2023-12-27T09:41:18.612+0545] {processor.py:161} INFO - Started process (PID=7520) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:41:18.615+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:41:18.616+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:41:18.616+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:41:18.622+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:41:18.621+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:41:18.623+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:41:18.643+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.036 seconds
[2023-12-27T09:41:58.679+0545] {processor.py:161} INFO - Started process (PID=7735) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:41:58.682+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:41:58.684+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:41:58.683+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:41:58.690+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:41:58.689+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:41:58.691+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:41:58.717+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.040 seconds
[2023-12-27T09:42:34.848+0545] {processor.py:161} INFO - Started process (PID=7802) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:42:34.850+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:42:34.851+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:42:34.850+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:42:34.853+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:42:34.852+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:42:34.853+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:42:34.866+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.021 seconds
[2023-12-27T09:43:07.855+0545] {processor.py:161} INFO - Started process (PID=7846) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:43:07.857+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:43:07.859+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:43:07.858+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:43:07.870+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:43:07.867+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:43:07.871+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:43:07.896+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.043 seconds
[2023-12-27T09:43:43.976+0545] {processor.py:161} INFO - Started process (PID=7898) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:43:43.978+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:43:43.979+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:43:43.978+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:43:43.983+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:43:43.982+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:43:43.983+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:43:43.999+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.026 seconds
[2023-12-27T09:44:23.526+0545] {processor.py:161} INFO - Started process (PID=7949) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:44:23.528+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:44:23.529+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:44:23.529+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:44:23.531+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:44:23.530+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:44:23.531+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:44:23.544+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.019 seconds
[2023-12-27T09:44:58.036+0545] {processor.py:161} INFO - Started process (PID=7997) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:44:58.037+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:44:58.039+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:44:58.038+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:44:58.047+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:44:58.045+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:44:58.048+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:44:58.068+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.039 seconds
[2023-12-27T09:45:36.485+0545] {processor.py:161} INFO - Started process (PID=8430) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:45:36.487+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:45:36.488+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:45:36.487+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:45:36.490+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:45:36.490+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:45:36.490+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:45:36.508+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.026 seconds
[2023-12-27T09:46:12.104+0545] {processor.py:161} INFO - Started process (PID=8484) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:46:12.111+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:46:12.112+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:46:12.112+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:46:12.118+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:46:12.116+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:46:12.119+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:46:12.202+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.102 seconds
[2023-12-27T09:46:48.375+0545] {processor.py:161} INFO - Started process (PID=8521) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:46:48.377+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:46:48.378+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:46:48.378+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:46:48.383+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:46:48.382+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:46:48.384+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:46:48.405+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.035 seconds
[2023-12-27T09:47:27.211+0545] {processor.py:161} INFO - Started process (PID=8560) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:47:27.214+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:47:27.215+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:47:27.214+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:47:27.221+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:47:27.219+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:47:27.221+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:47:27.241+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.034 seconds
[2023-12-27T09:48:05.699+0545] {processor.py:161} INFO - Started process (PID=8622) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:48:05.702+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:48:05.703+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:48:05.703+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:48:05.708+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:48:05.706+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:48:05.709+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:48:05.726+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.030 seconds
[2023-12-27T09:48:44.612+0545] {processor.py:161} INFO - Started process (PID=8666) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:48:44.615+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:48:44.618+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:48:44.617+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:48:44.624+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:48:44.623+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:48:44.625+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:48:44.642+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.035 seconds
[2023-12-27T09:49:19.482+0545] {processor.py:161} INFO - Started process (PID=8704) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:49:19.484+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:49:19.486+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:49:19.485+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:49:19.490+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:49:19.489+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:49:19.491+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:49:19.510+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.032 seconds
[2023-12-27T09:49:56.727+0545] {processor.py:161} INFO - Started process (PID=8743) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:49:56.729+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:49:56.729+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:49:56.729+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:49:56.732+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:49:56.731+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:49:56.732+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:49:56.747+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.023 seconds
[2023-12-27T09:50:35.571+0545] {processor.py:161} INFO - Started process (PID=8796) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:50:35.573+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:50:35.574+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:50:35.574+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:50:35.576+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:50:35.576+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:50:35.577+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:50:35.590+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.021 seconds
[2023-12-27T09:51:15.247+0545] {processor.py:161} INFO - Started process (PID=8871) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:51:15.249+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:51:15.250+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:51:15.250+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:51:15.253+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:51:15.253+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:51:15.254+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:51:15.269+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.026 seconds
[2023-12-27T09:51:54.137+0545] {processor.py:161} INFO - Started process (PID=8917) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:51:54.139+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:51:54.140+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:51:54.140+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:51:54.145+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:51:54.144+0545] {dagbag.py:348} ERROR - Failed to import: //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
Traceback (most recent call last):
  File "/home/sarita/Documents/apache-airflow-project/.venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "//home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py", line 6, in <module>
    from db_connection import BATCH_SIZE, get_sqlite_connection, get_clickhouse_connection, SQLITE_TABLE
ModuleNotFoundError: No module named 'db_connection'
[2023-12-27T09:51:54.145+0545] {processor.py:842} WARNING - No viable dags retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:51:54.165+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.033 seconds
[2023-12-27T09:52:13.066+0545] {processor.py:161} INFO - Started process (PID=8975) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:52:13.067+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:52:13.068+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:52:13.068+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:52:13.109+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:52:13.182+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:52:13.182+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T09:52:13.202+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:52:13.202+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-08-03T00:00:00+00:00, run_after=2023-08-04T00:00:00+00:00
[2023-12-27T09:52:13.229+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.166 seconds
[2023-12-27T09:53:35.109+0545] {processor.py:161} INFO - Started process (PID=9107) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:53:35.110+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:53:35.111+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:53:35.110+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:53:35.132+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:53:35.142+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:53:35.142+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T09:53:35.159+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:53:35.159+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-08-17T00:00:00+00:00, run_after=2023-08-18T00:00:00+00:00
[2023-12-27T09:53:35.177+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.070 seconds
[2023-12-27T09:55:49.610+0545] {processor.py:161} INFO - Started process (PID=9303) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:55:49.612+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T09:55:49.613+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:55:49.613+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:55:49.642+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T09:55:49.725+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:55:49.725+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T09:55:49.743+0545] {logging_mixin.py:188} INFO - [2023-12-27T09:55:49.742+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-08-30T00:00:00+00:00, run_after=2023-08-31T00:00:00+00:00
[2023-12-27T09:55:49.760+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.152 seconds
[2023-12-27T12:10:02.866+0545] {processor.py:161} INFO - Started process (PID=12451) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:10:02.868+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:10:02.869+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:10:02.868+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:10:02.895+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:10:02.971+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:10:02.971+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:10:02.987+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:10:02.987+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-05-22T00:00:00+00:00, run_after=2023-05-23T00:00:00+00:00
[2023-12-27T12:10:03.007+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.143 seconds
[2023-12-27T12:10:56.152+0545] {processor.py:161} INFO - Started process (PID=12495) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:10:56.153+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:10:56.154+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:10:56.154+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:10:56.180+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:10:56.264+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:10:56.264+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:10:56.283+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:10:56.282+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-05-24T00:00:00+00:00, run_after=2023-05-25T00:00:00+00:00
[2023-12-27T12:10:56.301+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.150 seconds
[2023-12-27T12:13:49.377+0545] {processor.py:161} INFO - Started process (PID=12702) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:13:49.379+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:13:49.380+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:13:49.379+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:13:49.409+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:13:49.489+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:13:49.488+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:13:49.504+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:13:49.504+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-06-01T00:00:00+00:00, run_after=2023-06-02T00:00:00+00:00
[2023-12-27T12:13:49.525+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.150 seconds
[2023-12-27T12:19:26.677+0545] {processor.py:161} INFO - Started process (PID=13630) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:19:26.677+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:19:26.678+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:19:26.678+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:19:26.701+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:19:26.711+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:19:26.710+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:19:26.726+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:19:26.726+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-07-04T00:00:00+00:00, run_after=2023-07-05T00:00:00+00:00
[2023-12-27T12:19:26.744+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.070 seconds
[2023-12-27T12:22:57.065+0545] {processor.py:161} INFO - Started process (PID=13936) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:22:57.066+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:22:57.066+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:22:57.066+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:22:57.093+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:22:57.176+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:22:57.175+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:22:57.194+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:22:57.194+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-07-18T00:00:00+00:00, run_after=2023-07-19T00:00:00+00:00
[2023-12-27T12:22:57.222+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.159 seconds
[2023-12-27T12:33:45.747+0545] {processor.py:161} INFO - Started process (PID=14787) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:33:45.747+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:33:45.748+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:33:45.748+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:33:45.774+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:33:45.851+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:33:45.851+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:33:45.867+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:33:45.867+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-07-26T00:00:00+00:00, run_after=2023-07-27T00:00:00+00:00
[2023-12-27T12:33:45.885+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.140 seconds
[2023-12-27T12:35:03.162+0545] {processor.py:161} INFO - Started process (PID=14847) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:35:03.163+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:35:03.163+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:35:03.163+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:35:03.188+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:35:03.267+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:35:03.266+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:35:03.284+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:35:03.284+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-07-27T00:00:00+00:00, run_after=2023-07-28T00:00:00+00:00
[2023-12-27T12:35:03.301+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.141 seconds
[2023-12-27T12:35:56.337+0545] {processor.py:161} INFO - Started process (PID=15090) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:35:56.338+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:35:56.339+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:35:56.338+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:35:56.361+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:35:56.374+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:35:56.374+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:35:56.389+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:35:56.388+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-07-28T00:00:00+00:00, run_after=2023-07-29T00:00:00+00:00
[2023-12-27T12:35:56.415+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.079 seconds
[2023-12-27T12:38:50.888+0545] {processor.py:161} INFO - Started process (PID=15321) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:38:50.889+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:38:50.890+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:38:50.889+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:38:50.921+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:38:51.074+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:38:51.074+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:38:51.085+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:38:51.085+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to None, run_after=None
[2023-12-27T12:38:51.110+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.223 seconds
[2023-12-27T12:39:56.852+0545] {processor.py:161} INFO - Started process (PID=15557) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:39:56.855+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:39:56.857+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:39:56.856+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:39:56.895+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:39:56.969+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:39:56.969+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:39:56.986+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:39:56.986+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-08-01T00:00:00+00:00, run_after=2023-08-02T00:00:00+00:00
[2023-12-27T12:39:57.009+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.164 seconds
[2023-12-27T12:42:11.919+0545] {processor.py:161} INFO - Started process (PID=16664) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:42:11.920+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:42:11.921+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:42:11.920+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:42:11.948+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:42:11.958+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:42:11.958+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:42:11.974+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:42:11.973+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-08-16T00:00:00+00:00, run_after=2023-08-17T00:00:00+00:00
[2023-12-27T12:42:11.991+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.074 seconds
[2023-12-27T12:43:52.968+0545] {processor.py:161} INFO - Started process (PID=16862) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:43:52.968+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:43:52.969+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:43:52.969+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:43:52.993+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:43:53.073+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:43:53.072+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:43:53.090+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:43:53.090+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-08-30T00:00:00+00:00, run_after=2023-08-31T00:00:00+00:00
[2023-12-27T12:43:53.107+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.141 seconds
[2023-12-27T12:44:56.861+0545] {processor.py:161} INFO - Started process (PID=17028) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:44:56.862+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:44:56.863+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:44:56.862+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:44:56.888+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:44:57.019+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:44:57.019+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:44:57.049+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.189 seconds
[2023-12-27T12:52:06.410+0545] {processor.py:161} INFO - Started process (PID=17570) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:52:06.410+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:52:06.411+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:52:06.410+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:52:06.438+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:52:06.575+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:52:06.575+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:52:06.607+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.200 seconds
[2023-12-27T12:52:58.912+0545] {processor.py:161} INFO - Started process (PID=17648) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:52:58.914+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:52:58.915+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:52:58.915+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:52:58.943+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:52:58.952+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:52:58.952+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:52:58.968+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:52:58.968+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-12-28T00:00:00+00:00, run_after=2023-12-29T00:00:00+00:00
[2023-12-27T12:52:58.989+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.079 seconds
[2023-12-27T12:53:38.862+0545] {processor.py:161} INFO - Started process (PID=17766) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:53:38.864+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:53:38.866+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:53:38.865+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:53:38.905+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:53:38.969+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:53:38.969+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:53:38.982+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:53:38.982+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-09-22T00:00:00+00:00, run_after=2023-09-23T00:00:00+00:00
[2023-12-27T12:53:39.000+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.146 seconds
[2023-12-27T12:53:55.907+0545] {processor.py:161} INFO - Started process (PID=17784) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:53:55.908+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:53:55.908+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:53:55.908+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:53:55.934+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:53:55.939+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:53:55.938+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:53:56.047+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.142 seconds
[2023-12-27T12:57:36.642+0545] {processor.py:161} INFO - Started process (PID=18017) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:57:36.643+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T12:57:36.644+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:57:36.643+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:57:36.668+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T12:57:36.746+0545] {logging_mixin.py:188} INFO - [2023-12-27T12:57:36.746+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T12:57:36.779+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.139 seconds
[2023-12-27T16:15:19.059+0545] {processor.py:161} INFO - Started process (PID=24717) to work on //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T16:15:19.060+0545] {processor.py:830} INFO - Processing file //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py for tasks to queue
[2023-12-27T16:15:19.060+0545] {logging_mixin.py:188} INFO - [2023-12-27T16:15:19.060+0545] {dagbag.py:538} INFO - Filling up the DagBag from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T16:15:19.088+0545] {processor.py:840} INFO - DAG(s) 'metrics_pipeline' retrieved from //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py
[2023-12-27T16:15:19.171+0545] {logging_mixin.py:188} INFO - [2023-12-27T16:15:19.171+0545] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-27T16:15:19.190+0545] {logging_mixin.py:188} INFO - [2023-12-27T16:15:19.189+0545] {dag.py:3820} INFO - Setting next_dagrun for metrics_pipeline to 2023-09-26T00:00:00+00:00, run_after=2023-09-27T00:00:00+00:00
[2023-12-27T16:15:19.210+0545] {processor.py:183} INFO - Processing //home/sarita/Documents/apache-airflow-project/dags/taxi_metrics_dag.py took 0.153 seconds
